pages/dkp/kommander/2.0/index.md

```diff
@@ -5,7 +5,8 @@ title: Kommander 2.0
 version: 2.0
 menuWeight: 0
 subtree:
-beta: true
+  beta: true
```

Comment:
formatting
```suggestion
beta: true
```

Reply:
looking at https://github.com/mesosphere/dcos-docs-site/pull/3578/files I think these should be indented under `subtree:` ?

Reply:
You're right, you're right